### PR TITLE
[4.0] Adding some quickstart in default and alternate menu presets

### PR DESCRIPTION
**Pull request by infograf768**
### Summary of Changes
Quickstarts are added for Site Modules, Administrator modules and Custom Administrator Menus when using the alternative preset.

Quickstarts are added for Administrator modules and Custom Administrator Menus when using the default preset.

### Testing Instructions
Create a custom Administrator menu
Display the default Preset menu in the Admin menu module.
Then switch to the Alternative preset.

No  + sign (quickstart) for the items concerned

patch and test again. 

test clicking the + sign to confirm it does what it should.

### After patch
for both presets

For the custom administrator menu:
Screen Shot 2019-11-11 at 09 50 58

For the administrator modules
Screen Shot 2019-11-11 at 09 52 27

For the alternate preset is also added the quickstart for site modules
Screen Shot 2019-11-11 at 09 54 49

### CODE REVIEW

File: administrator/components/com_menus/presets/alternate.xml [hunk 1]
```diff
@@ -236,6 +236,8 @@
 				link="index.php?option=com_menus&amp;view=items&amp;menutype={sql:menutype}"
 				icon="{sql:icon}"
 				class="class:menu"
+				quicktask="index.php?option=com_menus&amp;task=item.add&amp;client_id=1&amp;menutype={sql:menutype}"
+
```
Comment (Quy) [inline administrator/components/com_menus/presets/alternate.xml]:
```suggestion
```

Comment (infograf768) [reply administrator/components/com_menus/presets/alternate.xml]: done

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on c43928441baacab8785f4cb7dc60683bd8b928cd<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/27045">issues.joomla.org/tracker/joomla-cms/27045</a>.</sub>

**Comment by SharkyKZ** [comment]
I have tested this item :white_check_mark: successfully on 91a59f42e60117eaadbc28f927cb3c447407875f<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/27045">issues.joomla.org/tracker/joomla-cms/27045</a>.</sub>

**Comment by SharkyKZ** [comment]
RTC.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/27045">issues.joomla.org/tracker/joomla-cms/27045</a>.</sub>

**Comment by wilsonge** [comment]
Thanks!

**Comment by brianteeman** [comment]
This PR broke the accessibility of the menu as we now have multiple links with the same name

**Comment by infograf768** [comment]
Can you give an example? And how to test accessibility here.

**Comment by brianteeman** [comment]
you have two links with the same name
"add new module"

**Comment by infograf768** [comment]
Easy enough to solve. We just need to create 2 new strings as I suppose that MOD_MENU_MENU_MANAGER_NEW_MENU_ITEM is also concerned.

On it now.

**Comment by infograf768** [comment]
@brianteeman
Please test https://github.com/joomla/joomla-cms/pull/27079

**Pull request merged**